packages/rum-core/src/bootstrap.js

```diff
@@ -23,19 +23,22 @@
  *
  */
 
-import { isPlatformSupported, patchAll } from '@elastic/apm-rum-core'
+import { isPlatformSupported } from './common/utils'
+import { patchAll } from './common/patching'
+import { bootstrapMetrics } from './performance-monitoring/metrics'
 
-var alreadyBootstrap = false
-var enabled = false
+let alreadyBootstrap = false
+let enabled = false
 
-export default function bootstrap() {
+export function bootstrap() {
   if (alreadyBootstrap) {
     return enabled
   }
   alreadyBootstrap = true
 
   if (isPlatformSupported()) {
     patchAll()
+    bootstrapMetrics()
```

Comment:
nit: can you name it something like `observeHidden` or `observeVisibility`? bootstrap doesn't even come close to what we are doing in that function.

Reply:
I think we should start using bootstrap (or something similar) to indicate that this function has side-effects!

Reply:
IMO, This function doesn't have side effects, we don't add anything to global, Adding listeners are not side effecty. If you refer to `metrics` its on module scope not on global.

Reply:
Why adding an event listener is not a side-effect? IMO, it's adding a global event therefore it's a side-effect.

Reply:
Observers are not side effect. Anyway its not a hard concern for this PR.

Reply:
I can't say I agree with that, as an example consider the case in which bootstrap is called twice, in that case two event listeners are going to be registered which is obviously not a correct logic. the bootstrap function indicates that the operation within it should be executed once per application.